jetty-ee9/jetty-ee9-servlet/src/main/java/org/eclipse/jetty/ee9/servlet/DefaultServlet.java

```diff
@@ -308,6 +300,17 @@ public void init()
             LOG.debug("resource base = {}", _baseResource);
     }
 
+    private static ByteBufferPool getByteBufferPool(ContextHandler contextHandler)
```

Comment:
I think I saw the same piece of code duplicated in other classes. Can't it be factored out?

Reply:
Don't thinks so, its duplicated across ee9 and ee10 which also use different `ContextHandler` classes.